### PR TITLE
Wargaming.WorldOfWarshipsModStation version 2.1.0.2

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34419)

### CODE REVIEW

File: manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation
+PackageVersion: 2.1.0.2
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.installer.yaml:7]: It looks like this program installs to `C:\Windows\Program Files` and not `C:\Windows\Program Files (x86)`
```suggestion
- Architecture: neutral
```

File: manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation
```
Comment (KaranKad) [inline manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Wargaming.WorldOfWarshipsModStation
```

File: manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation
```
Comment (KaranKad) [inline manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.installer.yaml:4]:
```suggestion
PackageIdentifier: Wargaming.WorldOfWarshipsModStation
```

File: manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation
```
Comment (KaranKad) [inline manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.yaml:4]:
```suggestion
PackageIdentifier: Wargaming.WorldOfWarshipsModStation
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml (hunk 2 above)
Comment (KaranKad) [reply manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml:4]: Make sure to change `manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml` to `manifests/w/Wargaming/WorldOfWarshipsModStation/2.1.0.2/Wargaming.WorldOfWarshipsModStation.yaml` for each yaml file

**Pull request closed without merging**